client/src/components/layouts/learn.css

```diff
@@ -2,22 +2,14 @@
   height: calc(100vh - var(--header-height));
 }
 
-#learn-app-wrapper .reflex-container.vertical .reflex-splitter:hover {
```

Comment:
@ahmadabdolsaheb Could you remove this file from this PR. These changes have some issues and I fixed it in this PR #36736

Reply:
@ValeraS sure :)